sky/serve/replica_managers.py

```diff
@@ -434,13 +449,15 @@ def __init__(self, service_name: str,
         self.lock = threading.Lock()
         self._next_replica_id: int = 1
         self._service_name: str = service_name
-        self._readiness_path: str = spec.readiness_path
-        self._initial_delay_seconds: int = spec.initial_delay_seconds
-        self._post_data: Optional[Dict[str, Any]] = spec.post_data
         self._uptime: Optional[float] = None
-        logger.info(f'Readiness probe path: {self._readiness_path}\n'
-                    f'Initial delay seconds: {self._initial_delay_seconds}\n'
-                    f'Post data: {self._post_data}')
+        logger.info(f'Readiness probe path: {spec.readiness_path}\n'
+                    f'Initial delay seconds: {spec.initial_delay_seconds}\n'
+                    f'Post data: {spec.post_data}')
+
+        self.latest_version: int = serve_constants.INITIAL_VERSION
+        self.oldest_version: int = serve_constants.INITIAL_VERSION
```

Comment:
What's the difference between `latest_version` and `oldest_version`. Can we add a comment for the prupose of the two different variables?

Maybe `oldest_version` -> `replica_least_recent_version`